app/models/course/lesson_plan/item.rb

```diff
@@ -3,7 +3,7 @@ class Course::LessonPlan::Item < ApplicationRecord
   include Course::LessonPlan::ItemTodoConcern
 
   actable optional: true
-  has_many_attachments
+  has_many_attachments on: :description
```

Comment:
Similarly, about 800 lesson plan items already have attachments. Will they be affected by this change?